knowledge-base/adr/outdated/0001-circuit-breaker-options.md

```diff
@@ -1,5 +1,8 @@
 # Circuit Breaker Options
 
+
+status: **superseded** [we will add resilience via middleware approach](../0031-resilience-options.md)
```

Comment:
```suggestion
Status: **superseded** 

We will add resilience via middleware approach ([ADR#0031](../0031-resilience-options.md)).
```

[pp] I think if something like this format was used consistently, it would look a bit nicer 😊